Pods/SBTUITestTunnelHost/README.md

```diff
@@ -22,9 +22,20 @@ Add files in the *SBTUITestTunnelHost* to the UI test target.
 
 ## Usage
 
-### Mac
+### Mac Server App
 
-Launch the Mac App (either by compiling the `Mac Host/SBTUITunnelHostServer.xcworkspace` or launching the executable in `Mac Host/Binary/SBTUITestTunnelServer.zip`) which will fire a server on your local machine on port 8667. The current status of the server will be shown in the macOS menubar.
+Launch the Mac App (either by compiling the `SBTUITunnelHostServer/SBTUITunnelHostServer.xcworkspace` or launching the executable in `SBTUITunnelHostServer/Binary/SBTUITestTunnelServer.zip`) which will fire a server on your local machine on port 8667. The current status of the server will be shown in the macOS menubar.
+
+#### Executable under Pods folder
```

Comment:
Today I've made a few changes to the host server app to respond to this [issue](https://github.com/Subito-it/SBTUITestTunnelHost/issues/11). To reduce maintenance I decided to remove the binary from the repo and let the user build the server app by themself which shouldn't be a problem since I added a simple script that does all the work. As a consequence the following installation instruction will no longer be working 😕.

Reply:
Okay thank you. Since the two main concerns (README documentation and the pod source) are resolved I think it’s best to close this PR and keep relying on the surrounding documentation of [the main tunnel host project](https://github.com/Subito-it/SBTUITestTunnelHost/) to provide the full context and usage.